src/AerospikeCache.php

```diff
@@ -0,0 +1,133 @@
+<?php
+declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use Aerospike;
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+use Symfony\Component\Cache\Exception\CacheException;
+
+class AerospikeCache extends AbstractAdapter
+{
+    private const WRAPPER_NAME = 'data';
+
+    /** @var Aerospike */
+    private $aerospike;
+
+    /** @var string $aerospikeNamespace */
```

Comment:
`$aerospikeNamespace` should not be there (and if so, it should name same as the field)

also at #20 where is `$set` in annotation

Reply:
You are right. Removed the names of variables